.github/workflows/default.yml

```diff
@@ -91,7 +91,7 @@ jobs:
     - uses: actions-rs/cargo@v1
       with:
         command: test
-        args: --no-default-features
+        args: --no-default-features --features=cty
```

Comment:
Does it mean we have to enable feature cty in order to compile in no_std env?

Reply:
technically it's orthogonal to the `libc` feature, where `libc` has the appropriate types for the platform you could use that, where it doesn't you'll need `cty`

Reply:
Do we need libc at all (except for tests in src/utils.rs)?

Reply:
possibly not? but it's also normal to use libc when std is available

Reply:
Personally, I would also prefer a simpler dependency structure, e.g. always use `cty` instead of `libc` and have `libc` only as a dev-dependency if it is required for test code.

Reply:
> Personally, I would also prefer a simpler dependency structure, e.g. always use cty instead of libc and have libc only as a dev-dependency if it is required for test code.

Yes